dev/sql/schema_relational.sql

```diff
@@ -48,26 +59,61 @@ begin
 end;
 $not_null_trigger$ language plpgsql;
 
-CREATE FUNCTION notify_modified_models() RETURNS trigger AS $notify_trigger$
+CREATE FUNCTION log_modified_models() RETURNS trigger AS $log_notify_trigger$
 DECLARE
-    channel TEXT;
+    operation TEXT;
     payload TEXT;
 BEGIN
-    channel:= LOWER(TG_OP);
+    operation := LOWER(TG_OP);
     payload := TG_TABLE_NAME || '/' || NEW.id;
     IF (TG_OP = 'DELETE') THEN
         payload = TG_TABLE_NAME || '/' || OLD.id;
     END IF;
 
-    PERFORM pg_notify(channel, payload);
-    INSERT INTO os_notify_log_t (channel, payload, xact_id, timestamp) VALUES (channel, payload, pg_current_xact_id(), 'now');
+    INSERT INTO os_notify_log_t (operation, payload, xact_id, timestamp) VALUES (operation, payload, pg_current_xact_id(), 'now');
+    RETURN NULL;  -- AFTER TRIGGER needs no return
+END;
+$log_notify_trigger$ LANGUAGE plpgsql;
+
+CREATE FUNCTION notify_modified_models() RETURNS trigger AS $notify_trigger$
+DECLARE
+    pl TEXT;
+    body_content_text TEXT;
+BEGIN
+    -- Running the trigger for the first time in a transaction the table is created and dropped after commiting the transaction.
+    -- Every next run of the trigger in this transaction raises a notice that teh table exists. This is not ideal.
+    CREATE LOCAL TEMPORARY UNLOGGED TABLE
```

Comment:
When applying the schema, I get an error in this like:

`ERROR: syntax error at or near "UNLOGGED" (SQLSTATE 42601)`

On the Postgres Docu, it seems, that `TEMPORARY` and `UNLOGGED` can not be used together. There is a `|` in the documentation between them.

I don't have experience with either of the attributes. But it seems, that a temporary table is also unlogged. So the `UNLOGGED` attribute can be removed:

https://stackoverflow.com/a/59247206

There is a typo in the docstring above this line `teh` => `the`

Reply:
Done. Added the `UNLOGGED` shortly before going into weekend. ;)